### PR TITLE
Remove labels.yaml

### PR DESCRIPTION
**Pull request by nikhita**
Update `labels.yaml` with latest labels.

Came across this while removing all "thirdpartyresource" occurrences.  Noticed that other labels also need update, so opened a new PR. :)

Edit: updated to remove the file entirely. 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: labels.yaml [hunk 1]
```diff
@@ -368,3 +386,5 @@ labels:
   color: ededed
 - name: triaged
   color: d455d0
+- name: ¯\_(ツ)_/¯
```
Comment (nikhita) [inline labels.yaml]: this is great! :smile:

Comment (xiangpengzhao) [reply labels.yaml]: I like this haha!
/shrug

**Comment by nikhita** [comment]
Hmm, I think we can actually remove this file as it is not serving any purpose.

**Comment by nikhita** [comment]
@BenTheElder Does sig-testing use this file somewhere?

**Comment by BenTheElder** [comment]
I don't think so, the label syncer uses a config map in test-infra, not sure about the existence of any other tools. cc @fejta

**Comment by fejta** [comment]
The current version of this file is https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml

**Comment by nikhita** [comment]
> The current version of this file is https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml

Removed the `labels.yaml` file from the root since the canonical source is in test-infra. PTAL.

**Comment by nikhita** [comment]
@BenTheElder @fejta lgty?

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
/assign @lavalamp 
for approval

**Comment by nikhita** [comment]
/assign @thockin 
for approval

**Pull request merged**